packages/wrangler/src/api/form_data.ts

```diff
@@ -1,72 +1,11 @@
 import type {
   CfWorkerInit,
   CfModuleType,
-  CfVariable,
   CfModule,
+  CfDurableObjectMigrations,
 } from "./worker.js";
 import { FormData, Blob } from "formdata-node";
 
-// Credit: https://stackoverflow.com/a/9458996
-function toBase64(source: BufferSource): string {
-  let result = "";
-  const buffer = source instanceof ArrayBuffer ? source : source.buffer;
-  const bytes = new Uint8Array(buffer);
-  for (let i = 0; i < bytes.byteLength; i++) {
-    result += String.fromCharCode(bytes[i]);
-  }
-  return btoa(result);
-}
-
-function toBinding(
-  name: string,
-  variable: CfVariable
-): Record<string, unknown> {
-  if (typeof variable === "string") {
-    return { name, type: "plain_text", text: variable };
-  }
-
-  if ("namespaceId" in variable) {
-    return {
-      name,
-      type: "kv_namespace",
-      namespace_id: variable.namespaceId,
-    };
-  }
-
-  if ("class_name" in variable) {
-    return {
-      name,
-      type: "durable_object_namespace",
-      class_name: variable.class_name,
-      ...(variable.script_name && {
-        script_name: variable.script_name,
-      }),
-    };
-  }
-
-  const { format, algorithm, usages, data } = variable;
```

Comment:
@Electroid I had this code from your earlier prototype, but I don't see any of our code paths actually encounter it. Tbf I also don't understand how secrets work in detail, and the docs aren't helpful either. @petebacondarwin is probably going to have a look at secrets usage soon, but can you shed some light on how this is used?

Reply:
We can introduce this later, we never made it public with wrangler1. It allows you to import a `CryptoKey` as a binding.